.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,18 @@
+name: Question
+description: Ask a question for the Council
+labels: ["question", "S-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        You are welcome to ask a question for the Council here, though you may alternatively consider asking your question on the [Leadership Council Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/392734-council).
```

Comment:
I'm not sure we want to allow *any* questions here. Perhaps we can give more guidance on when Zulip and/or email would be much preferred?

Reply:
My intent is to make it easy to get in contact with the council. Not everyone is going to have a Zulip account, and some people don't want to set that up or to have another site or app to go to or keep open to have a discussion. 

I originally had a sentence that said something like "on Zulip, where you are likely to get more back-and-forth discussion.", but I didn't want to imply that a GitHub issue would more likely be ignored.

I'm not really expecting many (or any) questions to show up here, but I want to make sure that people can use whatever method they are comfortable with. I added something to that effect to try to clarify it.